script/blockchain/tezos.ts

```diff
@@ -28,53 +25,6 @@ function getChainValidatorsList(chain: string): ValidatorModel[] {
     return JSON.parse(readFileSync(`${(getChainValidatorsPath(chain))}/${validatorsList}`));
 }
```

Comment:
'getChainValidatorsList' is defined but never used

Reply:
@catenocrypt it's safe to remove? this one seems like a generic one might be used later

Reply:
feel free to remove, we no need to run any logic for validators here, it's manually updated